src/components/Search.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react'
 
-const Search = () => {
+const Search = (props) => {
   return (
     <div className="ui huge fluid icon input">
       <input
         type="text"
         placeholder={"Search your Recent Transactions"}
```

Comment:
I would also bind the value of this input to what you have on the state as well.